vscode/src/testController.ts

```diff
@@ -497,6 +524,11 @@ export class TestController {
       // components, so we want to show each individual test directory as a separate item
       const relativePath = vscode.workspace.asRelativePath(uri);
       const pathParts = relativePath.split(path.sep);
+
+      if (pathParts.some((part) => part === "fixtures")) {
```

Comment:
Could we limit the discovery to `*.rb` so we wouldn't need this check?

Reply:
The discovery is already only Ruby files, but in the LSP's own codebase we find Ruby file under fixtures because of the Prism submodule.